SWLOR.Game.Server/SWLOR.Game.Server.csproj

```diff
@@ -52,6 +52,7 @@
     <Reference Include="System.Data.Linq" />
     <Reference Include="System.Runtime.Serialization" />
     <Reference Include="System.Security" />
+    <Reference Include="System.Web.Extensions" />
```

Comment:
Another reminder that we need to remove this Web Extensions method that Visual Studio added for you.